packages/graphql-hooks-memcache/.size-snapshot.json

```diff
@@ -1,46 +1,46 @@
 {
   "lib/graphql-hooks-memcache.js": {
-    "bundled": 936,
-    "minified": 621,
-    "gzipped": 369
+    "bundled": 972,
+    "minified": 643,
+    "gzipped": 383
   },
   "es/graphql-hooks-memcache.js": {
-    "bundled": 763,
-    "minified": 490,
-    "gzipped": 311,
+    "bundled": 799,
+    "minified": 512,
+    "gzipped": 326,
     "treeshaked": {
       "rollup": {
-        "code": 45,
-        "import_statements": 45
+        "code": 46,
+        "import_statements": 46
       },
       "webpack": {
-        "code": 1062
+        "code": 1063
       }
     }
   },
   "es/graphql-hooks-memcache.mjs": {
```

Comment:
This looks good. Any idea why there's ~4x increased bundle size for `es/` and `dist/`?

Reply:
It seems that lru-cache has one dependency whereas tiny-lru has none.
I used one of the most used lru package I found, but maybe you have a better candidate ?

Reply:
No, that looks like a good candidate. I just wonder if it's possible to continue using `tiny-lru`. [This section](https://www.npmjs.com/package/tiny-lru#example) of their README describes how it should be imported or aliased.

The `graphql-hooks-memcache` tests pass with `import LRU from 'tiny-lru/lib/tiny-lru.es5.js'`, but I'm unable to whether this fixes the Webpack import issue.

Another option might be to try and alias `tiny-lru` in the rollup config, which would require `rollup-plugin-alias`.

Reply:
I tried the method described (that's why I link it in the initial PR description), but with no luck unfortunately, the Webpack import issue persisted. 

I am not really used to rollup, and was not sure if alias would work. I will try using this plugin, IMO adding a dev dependency is better than multiply the bundled size by 4.

Reply:
Ah, sorry I overlooked that link.

Same here. Thanks for taking a look. I wasn't sure of the best way to test the build with webpack. 🤞

Reply:
If ever you want to reproduce / play with this bug, you can clone the next.js / with-graphql-hooks example here: https://github.com/zeit/next.js/tree/canary/examples/with-graphql-hooks
```
npm i && npm run dev
```
will show you the error

Reply:
I am not sure if the new options is most optimal. It does bloat the bundle size quite a bit.

Reply:
@fabienheureux @hboylan - let me know what you think of #370 

I am also trying to get `tiny-lru` updated here https://github.com/avoidwork/tiny-lru/pull/27

Reply:
LGTM